### PR TITLE
Update prepare-for-directory-synchronization.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the UPN invalid characters with "Letters with diacritical marks are invalid characters, such as umlauts, accents, and tildes." as it is not just umlauts it is also accents and tildes.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3100

File: microsoft-365/enterprise/prepare-for-directory-synchronization.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ The attributes that you need to prepare are listed here:
   - Maximum number of characters for the domain name following the at sign (@): 48
   - Invalid characters: \ % &amp; \* + / = ? { } | \< \> ( ) ; : , [ ] " '
   - Characters allowed: A – Z, a - z, 0 – 9, ' . - _ ! # ^ ~
-  - An umlaut is also an invalid character.
+  - Letters with diacritical marks are invalid characters, such as umlauts, accents, and tildes.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/prepare-for-directory-synchronization.md]:
```suggestion
  - Letters with diacritical marks, such as umlauts, accents, and tildes, are invalid characters.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**